src/Resources/contao/library/Contao/Widget.php

```diff
@@ -263,6 +265,14 @@ public function __set($strKey, $varValue)
 				$this->strPrefix = $varValue;
 				break;
 
+			case 'useRawRequestData':
```

Comment:
Yeah not all attributes do unfortunately. So it's inconsistent already anyway but I added it.